purchase_batch_invoicing/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+========================
+Purchase Batch Invoicing
+========================
+
+This module extends the functionality of purchases to support batch invoicing
+purchase orders and to allow you to choose if you want them grouped by purchase
+order or by vendor.
+
+Configuration
+=============
+
+An automated task is included to invoice all pending purchase orders every
+week, but it is disabled by default. To enable it:
+
+#. Have *Administration / Settings* permissions.
+#. Go to *Settings > Technical > Automation > Scheduled Actions > Invoice all
+   pending purchase orders > Edit*.
+#. Enable *Active*.
```

Comment:
You might want to add:
#. Enable it by clicking on Active and set the date accordingly